pages/index.vue

```diff
@@ -1,5 +1,28 @@
-<!-- Remove these two lines once content is added -->
-<!-- eslint-disable-next-line vue/valid-template-root -->
-<template></template>
+<!-- eslint-disable vue/attribute-hyphenation -->
+<template>
+  <!-- long logo -->
+  <img class="mx-auto" src="../assets/logos/TransparentLongLogo.png" />
+  <!-- competitions -->
+  <h1 class="text-center font-montserrat text-4xl">Competitions</h1>
 
-<script setup></script>
+  <hr class="my-5 w-4/5 h-px mx-auto bg-black border-0" />
+
+  <div class="flex h-screen justify-center">
+    <div class="grid grid-cols-3 grid-rows-6 gap-6 justify-between">
```

Comment:
```suggestion
    <div class="mx-auto h-screen place-items-center justify-center grid grid-cols-3 grid-rows-6 gap-6">
```
This should remove the need for the above `div`.